src/lib/validationRules/remoteMatchingClientId/remoteMatchingClientId.ts

```diff
@@ -0,0 +1,78 @@
+//
+// Copyright 2022 Inrupt Inc.
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal in
+// the Software without restriction, including without limitation the rights to use,
+// copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the
+// Software, and to permit persons to whom the Software is furnished to do so,
+// subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED,
+// INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A
+// PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT
+// HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION
+// OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE
+// SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+//
+
+import { RemoteValidationContext, RemoteValidationRule } from "../../types";
+
+const remoteMatchingClientId: RemoteValidationRule = {
+  rule: {
+    type: "remote",
+    name: "Remote Document Client ID matches local one",
+    description:
+      "The remote Client Identifier URI and the local Client Identifier URI must match exactly.",
```

Comment:
I'm not sure what you call remote and local client Identifier URI ?

Reply:
clarified to:
``` ts
 name: "Client Identifier in remote document matches local one",
 description:
      "The remote Client Identifier URI specified in the remote document and the local Client Identifier URI must match exactly.",
```